classes/local_yuja/yuja_client.class.php

```diff
@@ -97,7 +97,8 @@ public function get_lti_params($course, $requestid) {
             'user_id' => $USER->id,
             'custom_context_id' => $course->idnumber,
             'custom_context_term' => $course->startdate,
-            'custom_plugin_info' => $this->get_plugin_info()
+            'custom_plugin_info' => $this->get_plugin_info(), 
+            'not_gradeable' => 'true'
```

Comment:
Download ngrok to test this locally. You should verify that not_gradeable is being properly passed in and parsed by the lti entry servlet